trl/trainer/reward_trainer.py

```diff
@@ -12,14 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import warnings
-from typing import Callable, Dict, List, Optional, Tuple, Union
+from typing import Any, Callable, Dict, List, Literal, Optional, Tuple, Union
```

Comment:
from the CI logs it seems that `Literal` is not available in `typing` for python<=3.7, can we change it with something else? 🙏

Reply:
You're lucky that it's not the [27th of June](https://endoflife.date/python) yet, or I would have refuse 😉
I'll get on it!